pkg/webhook/cspc.go

```diff
@@ -341,9 +341,13 @@ var (
 	// TODO: confirm all the compression types supported by control plane
 	// and update the map accordingly
 	SupportedCompression = map[string]bool{
-		"":    true,
-		"off": true,
-		"lz":  true,
+		"on":   true,
+		"off":  true,
+		"lzjb": true,
+		"gzip": true,
```

Comment:
We have to support values of `gzip-[1-9]` also since the data plane supports the same

Reply:
Command to create pool will be `zpool create <pool_name> <disk_path> -O compresstion=gzip-9`